src/x509/clu_request_setup.c

```diff
@@ -262,9 +261,41 @@ static int _wolfSSL_X509_extensions_print(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
 static int _wolfSSL_X509_REQ_attributes_print(WOLFSSL_BIO* bio,
         WOLFSSL_X509* x509, int indent)
 {
-    (void)bio;
-    (void)x509;
-    (void)indent;
+    WOLFSSL_X509_ATTRIBUTE* attr;
```

Comment:
Any status update on moving this function into wolfSSL proper?

Reply:
made a start to it here (https://github.com/wolfSSL/wolfssl/pull/5213)